sirepo/package_data/static/js/sirepo.js

```diff
@@ -158,6 +160,14 @@ SIREPO.app.factory('authState', function(appDataService, appState, errorService,
         controller.showWarning = true;
         controller.warningText = 'Server reported an error, please contact support@radiasoft.net.';
     };
+
+    self.premiumOrEnterprise = function() {
+        if (self.roles.indexOf('premium') >= 0) {
```

Comment:
This string should come from the server.

Reply:
https://github.com/radiasoft/sirepo/pull/2690/commits/6ae62e9e3521eeb67104be4c4eda58da7c49f4d4